### PR TITLE
web-ext 7.7.0 (new formula)

### PR DESCRIPTION
**Pull request by abitrolly**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

https://github.com/mozilla/web-ext

### CODE REVIEW

File: Formula/w/web-ext.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  test do
+    (testpath/"manifest.json").write <<~EOF
+      {
+        "manifest_version": 2,
+        "name": "minimal web extension",
+        "version": "0.0.1"
+      }
+    EOF
+    system "#{bin}/web-ext lint"
```
Comment (chenrui333) [inline Formula/w/web-ext.rb:25]: can we assert the response?

```suggestion
    system bin/"#{bin}/web-ext", "lint"
```

Comment (abitrolly) [reply Formula/w/web-ext.rb]: The output is this.
```
Validation Summary:

errors          0              
notices         0              
warnings        0
```
I am checking how to assert that.

Comment (abitrolly) [reply Formula/w/web-ext.rb]: Done.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @abitrolly a lot for your first contribution to homebrew-core!! 🎉

**Review by p-linnane — APPROVED** [review]

**Pull request merged**